stetho/src/main/java/com/facebook/stetho/inspector/protocol/ScreencastDispatcher.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.protocol;
+
+import android.app.Activity;
+import android.graphics.Bitmap;
+import android.os.Handler;
+import android.os.HandlerThread;
+import android.os.Looper;
+import android.util.Base64;
+import android.view.View;
+import com.facebook.stetho.common.LogUtil;
+import com.facebook.stetho.inspector.elements.android.ActivityTracker;
+import com.facebook.stetho.inspector.jsonrpc.JsonRpcPeer;
+import com.facebook.stetho.json.annotation.JsonProperty;
+
+import java.io.ByteArrayOutputStream;
+
+public class ScreencastDispatcher {
+  private final Handler mMainHandler = new Handler(Looper.getMainLooper());
+  private final HandlerThread mHandlerThread = new HandlerThread("Screencast Thread");
```

Comment:
Prefer an explicit init step that creates/starts the thread.  Constructors with side effects are often very hard to debug/predict.